.github/workflows/build.yml

```
@@ -41,48 +41,74 @@ jobs:
       - name: Run unit tests and generate coverage
-        run: dotnet-coverage collect "dotnet test VictoryCenter/VictoryCenter.UnitTests/VictoryCenter.UnitTests.csproj --configuration Release" -f xml -o "coverage.xml"
+        run: dotnet-coverage collect "dotnet test VictoryCenter/VictoryCenter.UnitTests/VictoryCenter.UnitTests.csproj --configuration Release" -f xml -o coverage1.xml
 
-      - name: Run integration tests and merge coverage
-        run: dotnet-coverage merge "coverage.xml" "dotnet-coverage collect \"dotnet test VictoryCenter/VictoryCenter.IntegrationTests/VictoryCenter.IntegrationTests.csproj --configuration Release\" -f xml -o \"coverage2.xml\"" -o "coverage.xml"
+      - name: Run integration tests and generate coverage
+        run: dotnet-coverage collect "dotnet test VictoryCenter/VictoryCenter.IntegrationTests/VictoryCenter.IntegrationTests.csproj --configuration Release" -f xml -o coverage2.xml
+
+      - name: Merge coverage reports
+        run: dotnet-coverage merge coverage1.xml coverage2.xml -o coverage.xml
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)